doc/ota_updates/ota_updates.md

```diff
@@ -318,7 +324,7 @@ Example header data:
     [HTTP_X_ESP8266_AP_MAC] => 1A:FE:AA:AA:AA:AA
     [HTTP_X_ESP8266_FREE_SPACE] => 671744
```

Comment:
Actually free space is the amount of memory we have between the sketch and SPIFFS. On a 4M device, SPIFFS starts at 1M mark. So this number looks correct.